[Update] WAAPI Post (#3747)
GoogleChrome/web.dev: src/site/content/en/blog/web-animations/index.md

---

> title: Using the full power of the Web Animations API

Reviewer: Even though I like the new title, I think she's right. The article is focused on Chrome 84.

The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative animations with JavaScript](https://www.youtube.com/watch?v=WaNoqBAp8NI). It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.

> While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), has already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in its browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.

Reviewer: ```suggestion
While [Firefox](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), have already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation), Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```

Reviewer: Also, can you please link to something official from Mozilla.

> While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), have already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in their browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.

Reviewer: ```suggestion
While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), has already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in its browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```

Reviewer: The Google standard is American English. I don't know why I didn't request this change before.
